### DISCUSSION TITLE
How do I buy an XTTS Commercial License since Coqui is shutting down?

### DISCUSSION THREAD

**Discussion by the-homeless-god** [coqui-ai/TTS#3711] [General Q&A]
Is it possible or doesn't make sense and I can use commercially it for myself or no?

**Comment by zombri-eats-brainz** [comment]
I would like to know the answer also. It sure would be nice to use this for voiceovers legally. But if the company has gone, who has the right to enforce the noncommercial license? Seems like a horrible waste if this project can never be anything more than a play toy.

**Reply by FrontierDK** [reply]
if you want to go the unsupported way, may I advice you to install everything in a virtual machine - and then create a good backup of that virtual machine. That will allow you to install and use later on, even if the hardware breaks down, this software source/mirror goes away etc.

If you need CUDA to train, look into PCI express passthrough [PCI express passthrough](https://devblogs.microsoft.com/scripting/passing-through-devices-to-hyper-v-vms-by-using-discrete-device-assignment/)
